package.json

```diff
@@ -57,6 +60,7 @@
     "@uniswap/governance": "^1.0.2",
     "@uniswap/liquidity-staker": "^1.0.2",
     "@uniswap/merkle-distributor": "1.0.1",
+    "@uniswap/redux-multicall": "^1.0.0",
```

Comment:
This should now go in devDeps, not deps, if it's to be used for widgets (ok to move in a later PR)

Reply:
reversed?